docs/dashboard/configuration/products.md

```diff
@@ -32,13 +32,15 @@ This is very important, as it means you are legally charging the correct amount
 
 1. Click _Create&nbsp;Product&nbsp;Type_ and fill out the General Information, Taxes, and Shipping cards.
 
-2. Click _Save_. An additional _Product Attributes_ card will appear. Assign previously created [attributes](dashboard/configuration/attributes.md) to your product type.
+2. Choose if the product type can be used to create shipping passes for your customers or can be used to create gift card type products that user will be able to pay with during checkout process.
```

Comment:
1. What shipping passes? I can't see any shipping passes radio button on the screenshot
2. The information implicates that dashboard-issued gift cards cannot be used to pay during checkout. Also user wants this product type to `create gift cards as products to be sold in their store`

Reply:
Not sure what happened. It seems there was an issue between my local version and the main one. This sentence shouldn't be there.